src/components/Transaction/Transaction.spec.js

```diff
@@ -0,0 +1,71 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Transaction from './';
+
+const change = 42.42;
+const date = '2022-07-07';
+const note = 'cloth';
+const category = 'hobbies and free time';
+const editTxt = 'Edit';
+const deleteTxt = 'Delete';
+
+describe('Transaction', () => {
+	it('should render correctly', () => {
+		render(
+			<Transaction change={change} date={date} note={note} category={category}>
+				<button>{editTxt}</button>
+				<button>{deleteTxt}</button>
+			</Transaction>
+		);
+		const changeTest = screen.getByText(change + ' €');
+		const dateTest = screen.getByText(date);
+		const noteTest = screen.getByText(note);
+		const categoryTest = screen.getByText(category);
+		const editButton = screen.getByText(editTxt);
+		const deleteButton = screen.getByText(deleteTxt);
```

Comment:
Try to use `getByRole` as often as possible. Especially for the buttons you can use: `screen.getByRole('button', {name: 'Edit'});`. It is a bit more specific and less prone to errors.